op-challenger/scripts/alphabet/charlie.sh

```diff
@@ -12,11 +12,12 @@ CHARLIE_KEY="74feb147d72bfae943e6b4e483410933d9e447d5dc47d52432dcc2c1454dabb7"
 MALLORY_ADDRESS="0x4641c704a6c743f73ee1f36C7568Fbf4b80681e4"
 MALLORY_KEY="28d7045146193f5f4eeb151c4843544b1b0d30a7ac1680c845a416fac65a7715"
 
-FAULT_GAME_ADDRESS="0x8daf17a20c9dba35f005b6324f493785d239719d"
+FAULT_GAME_ADDRESS=$(cat ../../.fault-game-address)
```

Comment:
maybe this could be passed in, but I'm not a huge fan of having a new hidden file with this configuration. It makes it a lot harder for people not very familiar with this code to run the demo.

Reply:
You should literally just be able to run "make alphabet" since the file is read by subsequent scripts.